sys/include/unaligned.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_unaligned unaligned access methods
+ * @ingroup     sys
+ * @brief       Provides functions for safe unaligned memory accesses
+ *
+ * This header provides functions to read values from pointers that are not
+ * necessarily aligned to the type's alignment requirements.
+ *
+ * E.g.,
+ *
+ *     uint16_t *foo = 0x123;
+ *     printf("%u\n", *foo);
+ *
+ * ... might cause an unaligned access, if uint16_t's is usually aligned ad
```

Comment:
rather prefer

```suggestion
 * ... might cause an unaligned access, if `uint16_t`'s is usually aligned ad
```

here.

Reply:
Also `s/ad$/at/` ;-)

Reply:
fixed (and removed 's)